lib/ripper-tags/json_formatter.rb

```diff
@@ -1,10 +1,22 @@
-require 'yajl'
+begin
+  require 'yajl/json_gem' unless defined?(::JSON)
```

Comment:
This handles the case where you might already have stdlib JSON loaded, but also have yajl-ruby gem present in the environment. Even though JSON might be already loaded and `JSON.dump` API available, without a conditional this would load YAJL's `JSON.dump` implementation and override stdlib one.